internal/build/info_test.go

```diff
@@ -1,9 +1,80 @@
 package build
 
 import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
 	"testing"
+
+	"os/exec"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/mod/modfile"
 )
 
 func TestGetDetails(t *testing.T) {
 	GetInfo()
 }
+
+// This test is stored in the internal package to avoid build errors since this
+// test itself is run during the CLI generation process.
```

Comment:
So this test cannot live inside `main_test.go` or packages touched by the CLI code generator because `go test` tries to compile the package the test is in and thus gets compilation errors. 

I see this comment is confusing because it implies internal is special in this context. Removed the command altogether.